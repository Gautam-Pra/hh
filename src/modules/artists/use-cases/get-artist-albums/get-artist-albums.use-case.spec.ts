import { beforeAll, describe, expect, test } from 'vitest'
import { GetArtistAlbumsUseCase } from '#modules/artists/use-cases'

describe('GetArtistAlbums', () => {
  let getArtistAlbumsUseCase: GetArtistAlbumsUseCase

  beforeAll(() => {
    getArtistAlbumsUseCase = new GetArtistAlbumsUseCase()
  })

  test('should get artist albums by artist id and return a list of albums', async () => {
    const albums = await getArtistAlbumsUseCase.execute({
      artistId: '1274170',
      page: 1,
      sortBy: 'popularity',
      sortOrder: 'asc'
    })

    expect(albums).toMatchSnapshot()
  })
})
