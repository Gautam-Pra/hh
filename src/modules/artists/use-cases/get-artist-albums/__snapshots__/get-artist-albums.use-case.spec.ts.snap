// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`GetArtistAlbums > should get artist albums by artist id and return a list of albums 1`] = `
{
  "albums": [
    {
      "artists": {
        "all": [
          {
            "id": "1274170",
            "image": [],
            "name": "Dua Lipa",
            "role": "singers",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/dua-lipa-songs/r-OWIKgpX2I_",
          },
        ],
        "featured": [],
        "primary": [
          {
            "id": "1274170",
            "image": [],
            "name": "Dua Lipa",
            "role": "primary_artists",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/dua-lipa-songs/r-OWIKgpX2I_",
          },
        ],
      },
      "description": "",
      "explicitContent": false,
      "id": "18530106",
      "image": [
        {
          "quality": "50x50",
          "url": "https://c.saavncdn.com/664/Don-t-Start-Now-Remixes--English-2020-20200107060932-50x50.jpg",
        },
        {
          "quality": "150x150",
          "url": "https://c.saavncdn.com/664/Don-t-Start-Now-Remixes--English-2020-20200107060932-150x150.jpg",
        },
        {
          "quality": "500x500",
          "url": "https://c.saavncdn.com/664/Don-t-Start-Now-Remixes--English-2020-20200107060932-500x500.jpg",
        },
      ],
      "language": "english",
      "name": "Don&#039;t Start Now (Remixes)",
      "playCount": 0,
      "songCount": 6,
      "type": "album",
      "url": "https://www.jiosaavn.com/album/dont-start-now-remixes/rIF8xV9lgDQ_",
      "year": 2020,
    },
    {
      "artists": {
        "all": [
          {
            "id": "573802",
            "image": [],
            "name": "Clarence Coffee Jr",
            "role": "music",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/clarence-coffee-jr-songs/2mX96rBxCYU_",
          },
          {
            "id": "1274170",
            "image": [],
            "name": "Dua Lipa",
            "role": "music",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/dua-lipa-songs/r-OWIKgpX2I_",
          },
          {
            "id": "566172",
            "image": [],
            "name": "Jeff Bhasker",
            "role": "music",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/jeff-bhasker-songs/lnl3ESr3WQo_",
          },
          {
            "id": "613919",
            "image": [],
            "name": "Ben Kohn",
            "role": "music",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/ben-kohn-songs/TktBXZIk6lY_",
          },
          {
            "id": "678029",
            "image": [],
            "name": "Camille Purcell",
            "role": "music",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/camille-purcell-songs/eikyrmwHNEY_",
          },
          {
            "id": "632102",
            "image": [],
            "name": "Pete Kelleher",
            "role": "music",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/pete-kelleher-songs/nYZUreZf,b8_",
          },
          {
            "id": "882822",
            "image": [],
            "name": "Shakka Philip",
            "role": "music",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/shakka-philip-songs/uuvVFJ5qi08_",
          },
          {
            "id": "613921",
            "image": [],
            "name": "Tom Barnes",
            "role": "music",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/tom-barnes-songs/ObvZUE45DcA_",
          },
          {
            "id": "707698",
            "image": [],
            "name": "Tove Lo",
            "role": "music",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/tove-lo-songs/wBPjTKDNv9Q_",
          },
          {
            "id": "591993",
            "image": [],
            "name": "Jason Evigan",
            "role": "music",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/jason-evigan-songs/,zVmyXajdpg_",
          },
          {
            "id": "702498",
            "image": [],
            "name": "Sarah Hudson",
            "role": "music",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/sarah-hudson-songs/pW4Y,NKAPPI_",
          },
          {
            "id": "599533",
            "image": [],
            "name": "Stephen Kozmeniuk",
            "role": "music",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/stephen-kozmeniuk-songs/lS1-2YuC5oc_",
          },
          {
            "id": "776593",
            "image": [],
            "name": "Caroline Ailin",
            "role": "music",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/caroline-ailin-songs/ksTT3ixnrvU_",
          },
          {
            "id": "577489",
            "image": [],
            "name": "Ian Kirkpatrick",
            "role": "music",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/ian-kirkpatrick-songs/zsvJSi5q-As_",
          },
          {
            "id": "672852",
            "image": [],
            "name": "Julia Michaels",
            "role": "music",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/julia-michaels-songs/OboBJBVTzgs_",
          },
          {
            "id": "4521858",
            "image": [],
            "name": "Samuel George Lewis",
            "role": "music",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/samuel-george-lewis-songs/ZVzkyvCfr-k_",
          },
          {
            "id": "2607495",
            "image": [],
            "name": "Sophie Frances Cooke",
            "role": "music",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/sophie-frances-cooke-songs/6TnITnwuIyw_",
          },
          {
            "id": "742265",
            "image": [],
            "name": "Chelcee Grimes",
            "role": "music",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/chelcee-grimes-songs/XwP3snpNdDI_",
          },
          {
            "id": "585225",
            "image": [],
            "name": "Crosby",
            "role": "music",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/crosby-songs/VnQRDtMVlow_",
          },
          {
            "id": "1130984",
            "image": [],
            "name": "Wallman",
            "role": "music",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/wallman-songs/QkFVfxVnRxw_",
          },
          {
            "id": "2100501",
            "image": [],
            "name": "Wartell",
            "role": "music",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/wartell-songs/BMNd28V2ydA_",
          },
          {
            "id": "566701",
            "image": [],
            "name": "Ali Tamposi",
            "role": "music",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/ali-tamposi-songs/uXzfkjl2h6g_",
          },
          {
            "id": "571060",
            "image": [],
            "name": "Andrew Farriss",
            "role": "music",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/andrew-farriss-songs/3eWWFeRs4ls_",
          },
          {
            "id": "1256966",
            "image": [],
            "name": "Andrew Wotman",
            "role": "music",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/andrew-wotman-songs/IKutWGGxYdI_",
          },
          {
            "id": "1825828",
            "image": [],
            "name": "Jordan K. Johnson",
            "role": "music",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/jordan-k.-johnson-songs/9iT4-qqOwIM_",
          },
          {
            "id": "567384",
            "image": [],
            "name": "Michael Hutchence",
            "role": "music",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/michael-hutchence-songs/T-MHZIru9-Q_",
          },
          {
            "id": "672854",
            "image": [],
            "name": "Stefan Johnson",
            "role": "music",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/stefan-johnson-songs/J2c,DBR4dso_",
          },
          {
            "id": "2419724",
            "image": [],
            "name": "David Charles Marshall Biral",
            "role": "music",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/david-charles-marshall-biral-songs/kbScH,rEjXM_",
          },
          {
            "id": "740123",
            "image": [],
            "name": "Denzel Michael-Akil Baptiste",
            "role": "music",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/denzel-michael-akil-baptiste-songs/vJVWPosaO6Q_",
          },
          {
            "id": "1290807",
            "image": [],
            "name": "Melanie Joy Fontana",
            "role": "music",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/melanie-joy-fontana-songs/CNTyLC,YPis_",
          },
          {
            "id": "7493251",
            "image": [],
            "name": "Michel Lindgren Schulz",
            "role": "music",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/michel-lindgren-schulz-songs/w-iIn-ycuSU_",
          },
          {
            "id": "4112470",
            "image": [],
            "name": "Taylor Upsahl",
            "role": "music",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/taylor-upsahl-songs/lB3-4Mrcy0Q_",
          },
          {
            "id": "633532",
            "image": [],
            "name": "Justin Tranter",
            "role": "music",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/justin-tranter-songs/MUdaah321CU_",
          },
          {
            "id": "1986316",
            "image": [],
            "name": "Kennedi",
            "role": "music",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/kennedi-songs/abSMdo,ClaQ_",
          },
          {
            "id": "2061563",
            "image": [],
            "name": "Angele",
            "role": "music",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/angele-songs/CTAPmcDBz3g_",
          },
          {
            "id": "570206",
            "image": [],
            "name": "Jacob Kasher Hindlin",
            "role": "music",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/jacob-kasher-hindlin-songs/anqSKuozjn0_",
          },
          {
            "id": "698397",
            "image": [],
            "name": "Emily Warren",
            "role": "music",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/emily-warren-songs/VwuRln-T5mE_",
          },
          {
            "id": "740039",
            "image": [],
            "name": "Scott Harris",
            "role": "music",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/scott-harris-songs/GD3O7jPuE30_",
          },
          {
            "id": "887068",
            "image": [],
            "name": "Sylvester Willy Sivertsen",
            "role": "music",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/sylvester-willy-sivertsen-songs/3jM1dmsjoPA_",
          },
          {
            "id": "1115484",
            "image": [],
            "name": "Jonathan Bellion",
            "role": "music",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/jonathan-bellion-songs/TKc04IZgCaw_",
          },
          {
            "id": "573803",
            "image": [],
            "name": "Marcus Lomax",
            "role": "music",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/marcus-lomax-songs/vHYBacETZwA_",
          },
          {
            "id": "1887754",
            "image": [],
            "name": "Michael Pollack",
            "role": "music",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/michael-pollack-songs/kdY6-WgNJ5M_",
          },
          {
            "id": "603634",
            "image": [],
            "name": "Miley Cyrus",
            "role": "music",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/miley-cyrus-songs/FRjYU7Dlr2U_",
          },
          {
            "id": "825030",
            "image": [],
            "name": "Clarence Coffee Jr.",
            "role": "music",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/clarence-coffee-jr.-songs/woRdMTkfuNE_",
          },
          {
            "id": "8811782",
            "image": [],
            "name": "Robin Oliver Frid",
            "role": "music",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/robin-oliver-frid-songs/U4bZtg6RZAQ_",
          },
          {
            "id": "714805",
            "image": [],
            "name": "Taylor Parks",
            "role": "music",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/taylor-parks-songs/-eh,usfVelo_",
          },
          {
            "id": "610392",
            "image": [],
            "name": "Uzoechi Emenike",
            "role": "music",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/uzoechi-emenike-songs/06XhH,faYHo_",
          },
          {
            "id": "601597",
            "image": [],
            "name": "Justin Parker",
            "role": "music",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/justin-parker-songs/Z2KwaTnBmuI_",
          },
          {
            "id": "657118",
            "image": [],
            "name": "Sam Dew",
            "role": "music",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/sam-dew-songs/NiEzi7IGpF4_",
          },
          {
            "id": "6552911",
            "image": [],
            "name": "Destin Choice Route",
            "role": "music",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/destin-choice-route-songs/iT2bdo2TfhM_",
          },
          {
            "id": "2436622",
            "image": [],
            "name": "DaBaby",
            "role": "music",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/dababy-songs/woXlnIzw1zU_",
          },
          {
            "id": "2122895",
            "image": [],
            "name": "Alejandro Borrero",
            "role": "music",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/alejandro-borrero-songs/DfnuyED5spc_",
          },
          {
            "id": "3792455",
            "image": [],
            "name": "Benito Antonio Martínez Ocasio",
            "role": "music",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/benito-antonio-martinez-ocasio-songs/suYOraOxkoI_",
          },
          {
            "id": "698785",
            "image": [],
            "name": "Daystar Peterson",
            "role": "music",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/daystar-peterson-songs/ag,k6,85t-w_",
          },
          {
            "id": "2122894",
            "image": [],
            "name": "Ivanni Rodriguez",
            "role": "music",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/ivanni-rodriguez-songs/D2uPsKsBtIk_",
          },
          {
            "id": "740056",
            "image": [],
            "name": "Jose Alvaro Osorio Balvin",
            "role": "music",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/jose-alvaro-osorio-balvin-songs/OAynqozci,E_",
          },
          {
            "id": "607510",
            "image": [],
            "name": "Marcos Masis",
            "role": "music",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/marcos-masis-songs/aDrbGbnsNAE_",
          },
          {
            "id": "1274170",
            "image": [],
            "name": "Dua Lipa",
            "role": "singers",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/dua-lipa-songs/r-OWIKgpX2I_",
          },
          {
            "id": "2061563",
            "image": [],
            "name": "Angele",
            "role": "singers",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/angele-songs/CTAPmcDBz3g_",
          },
          {
            "id": "603634",
            "image": [],
            "name": "Miley Cyrus",
            "role": "singers",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/miley-cyrus-songs/FRjYU7Dlr2U_",
          },
          {
            "id": "1017816",
            "image": [],
            "name": "J. Balvin",
            "role": "singers",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/j.-balvin-songs/kSQKdiFEeZ8_",
          },
          {
            "id": "2029479",
            "image": [],
            "name": "Bad Bunny",
            "role": "singers",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/bad-bunny-songs/hHU81hLRJc8_",
          },
          {
            "id": "1036632",
            "image": [],
            "name": "Tainy",
            "role": "singers",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/tainy-songs/Q-6FV2PFy6E_",
          },
        ],
        "featured": [],
        "primary": [
          {
            "id": "1274170",
            "image": [],
            "name": "Dua Lipa",
            "role": "primary_artists",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/dua-lipa-songs/r-OWIKgpX2I_",
          },
        ],
      },
      "description": "",
      "explicitContent": true,
      "id": "25198171",
      "image": [
        {
          "quality": "50x50",
          "url": "https://c.saavncdn.com/998/Future-Nostalgia-The-Moonlight-Edition-English-2021-20231110075215-50x50.jpg",
        },
        {
          "quality": "150x150",
          "url": "https://c.saavncdn.com/998/Future-Nostalgia-The-Moonlight-Edition-English-2021-20231110075215-150x150.jpg",
        },
        {
          "quality": "500x500",
          "url": "https://c.saavncdn.com/998/Future-Nostalgia-The-Moonlight-Edition-English-2021-20231110075215-500x500.jpg",
        },
      ],
      "language": "english",
      "name": "Future Nostalgia (The Moonlight Edition)",
      "playCount": 0,
      "songCount": 19,
      "type": "album",
      "url": "https://www.jiosaavn.com/album/future-nostalgia-the-moonlight-edition/WbwLTFwfrwI_",
      "year": 2021,
    },
    {
      "artists": {
        "all": [
          {
            "id": "1274170",
            "image": [],
            "name": "Dua Lipa",
            "role": "singers",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/dua-lipa-songs/r-OWIKgpX2I_",
          },
        ],
        "featured": [],
        "primary": [
          {
            "id": "1274170",
            "image": [],
            "name": "Dua Lipa",
            "role": "primary_artists",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/dua-lipa-songs/r-OWIKgpX2I_",
          },
        ],
      },
      "description": "",
      "explicitContent": false,
      "id": "16259227",
      "image": [
        {
          "quality": "50x50",
          "url": "https://c.saavncdn.com/675/Dua-Lipa-Deluxe-English-2017-20231110075309-50x50.jpg",
        },
        {
          "quality": "150x150",
          "url": "https://c.saavncdn.com/675/Dua-Lipa-Deluxe-English-2017-20231110075309-150x150.jpg",
        },
        {
          "quality": "500x500",
          "url": "https://c.saavncdn.com/675/Dua-Lipa-Deluxe-English-2017-20231110075309-500x500.jpg",
        },
      ],
      "language": "english",
      "name": "Dua Lipa (Deluxe)",
      "playCount": 0,
      "songCount": 17,
      "type": "album",
      "url": "https://www.jiosaavn.com/album/dua-lipa-deluxe/rxEvDmFrq0M_",
      "year": 2017,
    },
    {
      "artists": {
        "all": [
          {
            "id": "1331273",
            "image": [],
            "name": "Georgia Ku Overton",
            "role": "music",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/georgia-ku-overton-songs/,nkTPpLFYNk_",
          },
          {
            "id": "570816",
            "image": [],
            "name": "Giorgio Tuinfort",
            "role": "music",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/giorgio-tuinfort-songs/dzMqx-bmCd0_",
          },
          {
            "id": "698396",
            "image": [],
            "name": "Kyle Shearer",
            "role": "music",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/kyle-shearer-songs/aEnCHyqFhTI_",
          },
          {
            "id": "750432",
            "image": [],
            "name": "Martijn Garritsen",
            "role": "music",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/martijn-garritsen-songs/7oZ09GZC50c_",
          },
          {
            "id": "2095256",
            "image": [],
            "name": "Nathaniel Campany",
            "role": "music",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/nathaniel-campany-songs/t6,t3r50Hwk_",
          },
          {
            "id": "611692",
            "image": [],
            "name": "Andrew Jackson",
            "role": "music",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/andrew-jackson-songs/,pt92v4YXfE_",
          },
          {
            "id": "698397",
            "image": [],
            "name": "Emily Warren",
            "role": "music",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/emily-warren-songs/VwuRln-T5mE_",
          },
          {
            "id": "12760341",
            "image": [],
            "name": "Jamie 'Sermstyle' Sanderson",
            "role": "music",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/jamie-sermstyle-sanderson-songs/yRhUAYKVdZc_",
          },
          {
            "id": "1365555",
            "image": [],
            "name": "Philip Kembo",
            "role": "music",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/philip-kembo-songs/m,NJhlaFscw_",
          },
          {
            "id": "562807",
            "image": [],
            "name": "Sean Paul Henriques",
            "role": "music",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/sean-paul-henriques-songs/YEox-ir-av8_",
          },
          {
            "id": "1274170",
            "image": [],
            "name": "Dua Lipa",
            "role": "singers",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/dua-lipa-songs/r-OWIKgpX2I_",
          },
          {
            "id": "2518190",
            "image": [],
            "name": "BLACKPINK",
            "role": "singers",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/blackpink-songs/-r0by6dtVic_",
          },
          {
            "id": "565985",
            "image": [],
            "name": "Calvin Harris",
            "role": "singers",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/calvin-harris-songs/QMnR-zr4Bng_",
          },
          {
            "id": "1915946",
            "image": [],
            "name": "Silk City",
            "role": "singers",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/silk-city-songs/qLymIcWldyY_",
          },
          {
            "id": "677739",
            "image": [],
            "name": "Martin Garrix",
            "role": "singers",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/martin-garrix-songs/-v6U,o4czC8_",
          },
          {
            "id": "468845",
            "image": [],
            "name": "Sean Paul",
            "role": "singers",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/sean-paul-songs/GQPg5UvdL60_",
          },
        ],
        "featured": [],
        "primary": [
          {
            "id": "1274170",
            "image": [],
            "name": "Dua Lipa",
            "role": "primary_artists",
            "type": "artist",
            "url": "https://www.jiosaavn.com/artist/dua-lipa-songs/r-OWIKgpX2I_",
          },
        ],
      },
      "description": "",
      "explicitContent": false,
      "id": "16314225",
      "image": [
        {
          "quality": "50x50",
          "url": "https://c.saavncdn.com/213/Dua-Lipa-Complete-Edition-English-2017-20231110075257-50x50.jpg",
        },
        {
          "quality": "150x150",
          "url": "https://c.saavncdn.com/213/Dua-Lipa-Complete-Edition-English-2017-20231110075257-150x150.jpg",
        },
        {
          "quality": "500x500",
          "url": "https://c.saavncdn.com/213/Dua-Lipa-Complete-Edition-English-2017-20231110075257-500x500.jpg",
        },
      ],
      "language": "english",
      "name": "Dua Lipa (Complete Edition)",
      "playCount": 0,
      "songCount": 25,
      "type": "album",
      "url": "https://www.jiosaavn.com/album/dua-lipa-complete-edition/MBDzIIjqbHs_",
      "year": 2017,
    },
  ],
  "total": 24,
}
`;
